templates/start.html

```diff
@@ -29,7 +29,7 @@ <h1>Create in-toto layout
 <p>You will be asked to run a couple of in-toto commands while walking through this website, so make sure to have in-toto installed:</p>
 
 <pre class="code">
-pip install in-toto==0.1.1
+pip install -e git+https://github.com/in-toto/in-toto.git@v0.2.3.dev5#egg=in-totov0.2.3.dev5
```

Comment:
in-toto version

Reply:
I'd rather not instart things in an editable way, and not a dev version either.

Reply:
For that matter we probably don’t need to pin versions, moving forward?

Reply:
The editable install from GitHub was a temporary workaround, because back when I made this upgrade, we didn't have the latest substantial changes in PyPI yet.

Regarding pinning/auto-bump/no-pinning, I think that the layout tool is the use case, where pinning is the right thing to do (it's app vs. library). We don't have tests for this code, so we can't really tell if an update breaks the update or not.

Once we have in-toto 1.0.0, it should be okay to pin to the major version.

Reply:
> The editable install from GitHub was a temporary workaround, because back when I made this upgrade, we didn't have the latest substantial changes in PyPI yet.

Just realized that I it wouldn't have needed `-e` to install from GitHub.